### PR TITLE
Add Sonic Lost World

### PR DESCRIPTION
**Pull request by brothe2nd**
Adds Sonic Lost World.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/sonic-lost-world/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Sonic Lost World"
+description = "Sonic Lost World[5] is an action-adventure/platform video game developed by Sonic Team for the Wii U, Nintendo 3DS and Microsoft Windows. It was published by Nintendo in Europe and Australia and Sega in North America and Japan."
```
Comment (BreadFish64) [inline games/sonic-lost-world/game.dat]: Get rid of the [5]

File: games/sonic-lost-world/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "0004000000000000"
```
Comment (BreadFish64) [inline games/sonic-lost-world/game.dat]: I'm 97% certain that's not the actual title id

File: games/sonic-lost-world/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+region = "usa"
+release_date = "2013-10-29"
+
+[[testcases]]
+title = "0004000000000000"
```
Comment (BreadFish64) [inline games/sonic-lost-world/game.dat]: Same here

**Comment by BreadFish64** [comment]
Not sure if you need a screenshot if it doesn't even boot

File: games/sonic-lost-world/game.dat (hunk 2 above)
Comment (brothe2nd) [inline games/sonic-lost-world/game.dat]: Doesn't boot to get title id

**Comment by brothe2nd** [comment]
will not let me post without a screenshot

**Comment by Schplee** [comment]
@BreadFish64 At least 1 screenshot is mandatory.

File: games/sonic-lost-world/game.dat (hunk 2 above)
Comment (Schplee) [inline games/sonic-lost-world/game.dat]: @brothe2nd This is the USA TitleID: 00040000000c8c00

**Comment by Schplee** [comment]
LGTM - pending the change of the TitleIDs.

**Pull request merged**